### PR TITLE
fix: ensure we always update the git status correctly

### PR DESCRIPTION
**Pull request by jstrachan**
Signed-off-by: James Strachan <james.strachan@gmail.com>

### CODE REVIEW

File: pkg/cmd/create/create_step.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ var (
 		jx create step -sh "echo hello world"
 
 		# Creates a step on the command line: adding a pre step to the pullRequest promote lifecycle
-		jx create step -p pullRequest -l promote -m pre "echo before promote"
+		jx create step -p pullrequest -l promote -m pre "echo before promote"
```
Comment (martin-patsov) [inline pkg/cmd/create/create_step.go]:
```suggestion
		jx create step -p pullrequest -l promote -m pre -c "echo before promote"
```

**Comment by martin-patsov** [comment]
"echo before promote" without the `-c` parameter does nothing.

**Comment by jstrachan** [comment]
whoops - accidentally pushed a separate fix to this PR so gonna recreate

**Comment by jstrachan** [comment]
replaced with this PR https://github.com/jenkins-x/jx/pull/5250

**Pull request closed without merging**